src/App.js

```diff
@@ -1,29 +1,14 @@
-import logo from './logo.svg';
+/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/prefer-stateless-function */
 import './App.css';
+import React from 'react';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import Calculator from './components/calculator';
+
+class App extends React.Component {
```

Comment:
- [ ] Kindly convert the App component to a functional-based instead of class-based as per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20functional%20components).